tabpy-server/tabpy_server/version.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
```

Comment:
Same comment - I think this would be better served in __init__

Reply:
I moved the code to `tabpy.py` instead

Reply:
also fixed in the way you suggested it should be